ee/tabby-webserver/src/service/web_documents.rs

```diff
@@ -46,15 +46,18 @@ struct WebDocumentServiceImpl {
 impl WebDocumentService for WebDocumentServiceImpl {
     async fn list_custom_web_documents(
         &self,
+        ids: Option<Vec<ID>>,
         after: Option<String>,
         before: Option<String>,
         first: Option<usize>,
         last: Option<usize>,
     ) -> Result<Vec<CustomWebDocument>> {
         let (limit, skip_id, backwards) = graphql_pagination_to_filter(after, before, first, last)?;
+        let names: Option<Vec<String>> =
+            ids.map(|ids| ids.into_iter().map(|id| id.to_string()).collect::<Vec<_>>());
```

Comment:
For custom web documents, ids are not names.

https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/c6860d02c4fc17efbf513de74ef6ebc3766417ac/ee/tabby-webserver/src/service/web_documents.rs#L187

Reply:
Fixed